### PR TITLE
Update the documentation formatting

### PR DESCRIPTION
**Pull request by nanux**
Mainly changes in two pages for readability improvements, you can see the result here:
https://nanux.github.io/data-center-helm-charts/userguide/PREREQUISITES/
https://nanux.github.io/data-center-helm-charts/userguide/CONFIGURATION/

### CODE REVIEW

**Review by errcode1202 — APPROVED** [review]

File: docs/docs/userguide/CONFIGURATION.md [hunk 1]
```diff
@@ -29,8 +31,10 @@ Some key considerations to note when configuring the controller are:
     5. Service forwards request to appropriate pod
     6. Pod handles request 
 
+!!!info "Request body size"
+    By default the maximum allowed size for the request body is set to `250MB`. If the size in a request exceeds the maximum size of the client request body, an 413 error will be returned to the client. If the maximum request body can be configured by changing the value of `maxBodySize` in `values.yaml`.
```
Comment (errcode1202) [inline docs/docs/userguide/CONFIGURATION.md:35]: Can we wrap 413 in back ticks - `413`

File: docs/docs/userguide/PREREQUISITES.md [hunk 2]
```diff
@@ -4,38 +4,42 @@
 In order to deploy Atlassian’s Data Center products, the following is required:
 
 1. An understanding of [Kubernetes](https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/){.external} and [Helm](https://helm.sh/){.external} concepts
-2. A Kubernetes cluster, running Kubernetes `v1.19` or later
 3. `kubectl` `v1.19` or later, must be compatible with your cluster
 4. `helm` `v3.3` or later
```
Comment (errcode1202) [inline docs/docs/userguide/PREREQUISITES.md:8]: Can we hyperlink these two to their install instructions

**Review by tan-ro — APPROVED** [review]

**Review by errcode1202 — CHANGES_REQUESTED** [review]
Looks great! Just a couple of minor updates.

File: docs/docs/userguide/PREREQUISITES.md (hunk 2 above)
Comment (nanux) [reply docs/docs/userguide/PREREQUISITES.md]: Like this?

Screen Shot 2021-08-31 at 9 25 08 am

**Review by errcode1202 — APPROVED** [review]
🚀

**Pull request merged**